pelicun/resources/auto/Hazus_Earthquake_IM.py

```diff
@@ -89,18 +96,13 @@ def convertUnits(value, unit_in, unit_out):
         'mile': mile,
     }
     if (unit_in not in aval_types) or (unit_out not in aval_types):
-        print(
-            f"The unit {unit_in} or {unit_out} "
-            f"are used in auto_population but not supported"
-        )
-        return
-    value = value * scale_map[unit_in] / scale_map[unit_out]
-    return value
+        return None
```

Comment:
`print` statements are discouraged in production code. Users should have ultimate control over what is printed on the console. They should be able to issue their own print statements in their analysis scripts and see only the output they expected, nothing more. We already have a mechanism for logging and issuing warnings, so we should be using it there instead.
We can resurrect all removed print statements in this PR with a future PR, since this will need some small changes that wouldn't be considered "linting updates".

Reply:
I see why print statements are discouraged in production code and I agree with removing them from the core Pelicun engine. However, when it comes to misc files that we expect users to prepare, I would not be so strict. It is much easier to add print statements than to learn about logging and understand how it will show various messages. It also allows users to prepare these auto pop scripts independent of pelicun and use them in other environments effectively. The files will be part of the DLML repo anyway and I don't see a strong argument for purging print statements from every file there.

Having said all the above, I can be convinced, so please go ahead if you feel strongly about this.

Reply:
If `auto/Hazus_Earthquake_IM.py` is to be treated as a misc file and as a template for users to write other scripts to auto-populate their configuration files, then I see your point on allowing for mere `print` statements. We can relax that rule specifically for the `auto/` directory. That being said, if that script imported certain things and used them to log messages and issue warnings, users could pattern-match and adapt it to issue their own needs, or chose to just use `print`, so it seems to me we would be giving them more options. Either way is fine and easy to implement, so please let me know what you prefer.

Reply:
I see your point with providing a template. We already brought in unit conversion from pelicun, so we could bring additional features into the auto files. 

Logging is a bit tricky because the logger is part of the assessment, so we'd need to pass it from there like we do for the functions in file_io. If I were to create a template, I would do something like this:
```
msg = 'message to print'
if log:
  log.msg(msg)
else:
  print(msg)
```

I suggest the following strategy: for now, in this repo, let's put the print statements back in and set the auto directory to be skipped by ruff. In the DLML repo, create an issue that describes adding the logging and warning features from pelicun to the auto files that are provided by SimCenter to the library. It is important to make sure such behavior is not mandatory or expected for contributions coming from external users.

Reply:
Thanks, that's a great strategy. I restored the print statements.
1f36b108